src/main/java/study/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,54 @@
+package study.step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    static int splitAndSum(String text) {
+
+        if (text == null) return 0;
+        if (text.isEmpty()) return 0;
+
+        // java.util.regex 패키지의 Matcher, Pattern import
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(text);
```

Comment:
```suggestion
        Matcher matcher = Pattern.compile("//(.)\n(.*)").matcher(text);
```
변수명을 m으로 극단적으로 축약하는 것 보다 matcher라고 정확히 표현하는 것이 좋지 않을까요?